apps/web/src/components/RegisterForm/RegisterForm.tsx

```diff
@@ -36,56 +32,208 @@ export default function RegisterForm(): JSX.Element {
       formData.entries()
       // eslint-disable-next-line
     ) as any;
-    const { email, password } = formValue;
+    const { email, password, confirmPassword } = formValue;
 
     assert(recaptchaRef.current, 'ReCAPTCHA has not loaded');
-    const token = await recaptchaRef.current.executeAsync();
+    const token = recaptchaRef.current;
```

Comment:
Token will always be null now. I know we don't really use the token anyway but just wanted to call out that captcha doesn't execute and we aren't capturing the token now.